From e2a1c7f63003a6a80e861f6dc0b6c060a6d7385c Mon Sep 17 00:00:00 2001
From: Iain Lane <iainl@gnome.org>
Date: Sat, 16 Mar 2019 11:40:46 +0000
Subject: [PATCH] libgdm: Always de-duplicate

I put this inside an `#ifdef ENABLE_WAYLAND_SUPPORT` before, which would
mean that it's not called if that's not defined.
---
 libgdm/gdm-sessions.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libgdm/gdm-sessions.c b/libgdm/gdm-sessions.c
index 9001c47b..bcfe7658 100644
--- a/libgdm/gdm-sessions.c
+++ b/libgdm/gdm-sessions.c
@@ -302,11 +302,11 @@ collect_sessions (void)
         }
 
         g_array_free (wayland_search_array, TRUE);
+#endif
 
         g_hash_table_foreach_remove (gdm_available_sessions_map,
                                      remove_duplicate_sessions,
                                      names_seen_before);
-#endif
 }
 
 /**
-- 
2.18.1

